api/src/test/java/org/openmrs/module/dhisconnector/api/DHISConnectorServiceTest.java

```diff
@@ -51,6 +51,13 @@ public void transformToDHISPeriod() {
 		Assert.assertEquals("20171108", sdf.format(startDate.getTime()));
 		Assert.assertEquals("20171114", sdf.format(endDate.getTime()));
 		Assert.assertEquals(period, "2017WedW46");
+
+		//test weekly thursday, date now at 2017/nov/14
+		startDate = (Calendar) endDate.clone();
+		period = Context.getService(DHISConnectorService.class).transformToDHISPeriod(startDate, endDate, "WeeklyThursday", null);
```

Comment:
thanks @Piumal1999  have you also tested with a running instance of openmrs

Reply:
Yeah @HerbertYiga